beta/tests/tensorflow/test_transformations.py

```diff
@@ -71,7 +71,7 @@ def test_removal_command_union():
 
 def test_add_insertion_command_to_multiple_insertion_commands_same():
     check_fn = lambda src, dst: \
-        dst.type ==  TargetType.OPERATION_WITH_WEIGHTS and \
+        dst.type == TargetType.OPERATION_WITH_WEIGHTS and \
```

Comment:
I don't like the name `TargetType` because it does not have any sense to me at least. Can we come up with something more self-explainable? The previous name was better, IMHO.
NNCF code is getting more and more complicated. Providing self-explainable names to basic entities and classes is one way to simplify it.

Reply:
Let's discuss it offline and try to find a more simple and suitable solution.

Reply:
The problem here is that NNCF TF operation seems to be planning to use "removal" commands in addition to "insertion" operations, so `InsertionType` could no longer be used as a single general enum applicable both for PT and TF.

Reply:
Let revise it in the future.